shared/src/main/kotlin/org/jetbrains/kotlin/konan/exec/ExecuteCommand.kt

```diff
@@ -47,23 +48,23 @@ open class Command(initialCommand: List<String>) {
         return this
     }
 
-    open fun runProcess(): Int {
+    open fun runProcess() {
         val builder = ProcessBuilder(command)
 
         builder.redirectOutput(Redirect.INHERIT)
         builder.redirectInput(Redirect.INHERIT)
-        builder.redirectError(Redirect.INHERIT)
 
         val process = builder.start()
+
+        val reader = BufferedReader(InputStreamReader(process.errorStream))
```

Comment:
It seems that error stream is now will not be visible if process exits correctly. Where are all warnings printed? Into err or std out?

Reply:
Before nothing is seen if run with gradle plugin, only if run `konanc` you got all messages. I think how to save both modes.

Reply:
Warnings are printed in stderr